sdcm/nemesis.py

```diff
@@ -2055,7 +2055,7 @@ def disrupt_show_toppartitions(self):
         ks_cf_list = self.cluster.get_any_ks_cf_list(self.target_node)
         if not ks_cf_list:
             raise UnsupportedNemesis('User-defined Keyspace and ColumnFamily are not found.')
-        top_partition_api = random.choice([NewApiTopPartitionCmd, OldApiTopPartitionCmd])(ks_cf_list)
+        top_partition_api = OldApiTopPartitionCmd(ks_cf_list)
```

Comment:
Would be good to have comment here like "Enable it for 4.6" or something like this.

Reply:
On master it is enabled. When new branch will be created from master, it will be enabled